add DreamMaker_OverLord print info

leyap opened the pull request.

leyap committed:
add DreamMaker_OverLord print info

mleedavies commented on spark-print-data/printertypes.json:
```diff
@@ -15,89 +15,89 @@
 				"200x60_id": "data/ember_200x60.png",
 				"400x120_id": "data/ember_400x120.png"
 			},
-            "technology": "DLP",
```
a lot of whitespace issues in this makes it hard to check

mleedavies replied:
Can you something about the white space in this return,

leyap replied:
sorry, I can’t find white space

> 在 2015年11月12日，06:17，mleedavies notifications@github.com 写道：
> 
> In spark-print-data/printertypes.json https://github.com/spark3dp/print-manager/pull/3#discussion_r44595107:
> 
> > @@ -15,89 +15,89 @@
> >                 "200x60_id": "data/ember_200x60.png",
> >                 "400x120_id": "data/ember_400x120.png"
> >             },
> > -            "technology": "DLP",
> >   Can you something about the white space in this return,
> 
> —
> Reply to this email directly or view it on GitHub https://github.com/spark3dp/print-manager/pull/3/files#r44595107.

ghost commented:
Not sure we have a contributors agreement signed with you.

leyap committed:
fix white space

Our company is DFRobot. We sent the signed agreement last week.
Meanwhile the problem of white space   also fixed.

mleedavies commented on spark-print-data/printertypes.json:
```diff
@@ -656,10 +656,100 @@
+                {
+                    "type": "serial",
+                    "discovery": "serial",
+                    "protocol": "usb.serial",
```
Are you going to supply VID/PID for these connections https://github.com/spark3dp/print-manager/blob/master/spark-print-mgr/printers/drivers/driverConfig.json

ghost commented:
I add to revert this pull request as it did not run.
